experimental/examples/bubblebench/editable-shared-tree/src/Bubblebench.ts

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ISharedTree, SharedTreeFactory, singleTextCursor } from "@fluid-internal/tree";
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { AppState } from "./AppState";
+import { AppStateSchema, AppStateSchemaData, AppStateTreeProxy } from "./schema";
+
+export class Bubblebench extends DataObject {
+    public static get Name() {
+        return "@fluid-example/bubblebench-sharedtree";
+    }
+    private maybeTree?: ISharedTree = undefined;
+    private maybeAppState?: AppState = undefined;
+    static treeFactory = new SharedTreeFactory();
+
+    protected async initializingFirstTime() {
+        this.maybeTree = this.runtime.createChannel(
+            "unique-bubblebench-key-1337",
+            Bubblebench.treeFactory.type
+        ) as ISharedTree;
+
+        this.initializeTree(this.maybeTree);
+
+        // This line will fail with the error 0x17b /* "Channel to be binded should be in not bounded set" */);
```

Comment:
If this does fail, why do we do it and how do tests for it pass?

Reply:
Old comment, removed in latest commit.